scss/_patterns_navigation.scss

```diff
@@ -651,6 +682,152 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
   }
 
+  // SLIDING NAVIGATION
+  .p-navigation.is-sliding {
+    overflow-x: hidden; // stops overflow scrolling on the page on smaller screens
```

Comment:
Do we need overflow hidden both here and on p-navigation__nav below?

Reply:
So in order for the sliding animation to work properly, the the dropdown has to the visible ie: can not have `display: none`. This then causes the parent elements to overflow. That's why I added it to the parents of the sliding elements.